### PR TITLE
Translate more strings into spanish

### PR DESCRIPTION
**Pull request by rsf92**
I translated a few more strings. For some reason, it does not detect that [[BEGINNER_HINT_XX] ] is a link to another element of the pedia, it seems to happen only with my strings so there might be an error there, but I wasn't able to find it...

### CODE REVIEW

**Comment by geoffthemedio** [comment]
> it does not detect that [[BEGINNER_HINT_XX] ] is a link

The line `#*BEGINNER_HINT_15` string is commented out in your stringtable... But it would be just a macro subsitution anyway, not a clickable link.

**Comment by rsf92** [comment]
> > it does not detect that [[BEGINNER_HINT_XX] ] is a link
> 
> The line `#*BEGINNER_HINT_15` string is commented out in your stringtable... But it would be just a macro subsitution anyway, not a clickable link.
> 

Oh yeah makes sense. I thought it would display the English text instead.

I'll add the translation when I find some spare time

**Comment by geoffthemedio** [comment]
Substitutions with `[[KEY]]` are done within each stringtable when it is loaded. There is also code there that should be doing a fallback to english in any non-default stringtable as well, but it's possibly not working...?

Fallbacks to the english stringtable (or an error message about a missing string) are also done for keys that are looked up by C++ code, including if used in content scripts, at runtime as strings are displayed, after the stringtables are loaded.

Edit: I just tested by removing several planet size strings from the german stringtable, and the english equivalent strings showed up within the otherwise german text for the Gaia Transformation building. So I'm not sure why the beginner hint strings don't fall back to english for you...

Edit2: Fallbacks from spanish to english work for me with this pull request applied...
hints

File: default/stringtables/es.txt [hunk 1]
```diff
@@ -5108,8 +5108,8 @@ ENC_HOMEWORLDS
 #*ENC_VECTOR_TEXTURE_INFO
 #* <not translated 1970-01-01 01:00:00>
 
-#*ENC_INDEX
-#* <not translated 1970-01-01 01:00:00>
+ENC_INDEX
+Indice
```
Comment (Oberlus) [inline default/stringtables/es.txt]: Índice

**Review by Oberlus — CHANGES_REQUESTED** [review]

**Comment by rsf92** [comment]
> 
> Edit2: Fallbacks from spanish to english work for me with this pull request applied... )

I'll check again, but i'm pretty sure it wasn't working for me...

This is what i got on my machine...
Captura de pantalla de 2022-01-24 16-40-11

I'll try translating the hints just in case... I don't know

Okay, after translating a few strings it works for me. I'll finish translating the strings and then push the changes along with the little correction provided by @Oberlus

File: default/stringtables/es.txt (hunk 1 above)
Comment (rsf92) [reply default/stringtables/es.txt]: Should be fixed in this new commit

File: default/stringtables/es.txt [hunk 2]
```diff
@@ -5909,8 +5909,8 @@ Rango de detección
 #*HOMEWORLDS_TEXT
 #* <not translated 1970-01-01 01:00:00>
 
-#*FOCUS_TITLE
-#* <not translated 1970-01-01 01:00:00>
+FOCUS_TITLE
+Enfoque
```
Comment (Oberlus) [inline default/stringtables/es.txt]: I think Foco is better than Enfoque, but this is a tough one.

"Foco" is where you concentrate your attention or efforts, also the source of something, from where it expands.
"Enfoque" is more easily understood as point of view (how you look at something), camera focusing (to make image clearer), or shining a light on something.
So Enfoque is more about how (Enfoque industrial: I do this with the POV of an industrial businessman), and Foco is more about what (Foco en industria: concentrate efforst on industry).

File: default/stringtables/es.txt [hunk 3]
```diff
@@ -5919,8 +5919,8 @@ Rango de detección
 #* <not translated 1970-01-01 01:00:00>
 #* <not translated 1970-01-01 01:00:00>
 
-#*GROWTH_FOCUS_TITLE
-#* <not translated 1970-01-01 01:00:00>
+GROWTH_FOCUS_TITLE
+Enfoque de crecimiento
```
Comment (Oberlus) [inline default/stringtables/es.txt]: Foco en crecimiento

(I like that more than "Foco de crecimiento")

File: default/stringtables/es.txt [hunk 4]
```diff
@@ -6462,7 +6462,7 @@ BUILDING_TYPE
 Tipo de Edificio
 
 FOCUS
-Foco
+Enfoque
```
Comment (Oberlus) [inline default/stringtables/es.txt]: Foco

File: default/stringtables/es.txt [hunk 5]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
-#* <not translated 1970-01-01 01:00:00>
+[[BEGINNER_HINT_21]]
+
+[[BEGINNER_HINT_22]]'''
+
+RANDOM_BEGINNER_HINT
+Pista #%rawtext%
+
+BEGINNER_HINT_01
+1: Al empezar la partida, construir más [[predefinedshipdesign SD_SCOUT]]s puede facilitar explorar la región en cercana al mundo de origen.
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
1: Al empezar la partida, construir más [[predefinedshipdesign SD_SCOUT]]s puede facilitar explorar la región cercana al mundo de origen.
```

File: default/stringtables/es.txt [hunk 6]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_03
+3: Es posible reparar las naves en sistemas que tengan un [[buildingtype BLD_SHIPYARD_ORBITAL_DRYDOCK]] aliado si no se produjo combate en dicho sistema durante el turno anterior y el planeta tiene una [[metertype METER_HAPPINESS]] igual o mayor a 5. Las naves tendrán que esperar un turno en el sistema antes de que pueda empezar la reparación.
+
+BEGINNER_HINT_04
+4: Los planetas conectados por lineas de [[metertype METER_SUPPLY]] están vinculados y comparten puntos de producción (PP) de [[metertype METER_INDUSTRY]].
+
+BEGINNER_HINT_05
+5: Todos los sistemas comparten punto de [[metertype METER_RESEARCH]] (PI) que generan incluso sin tener líneas de [[metertype METER_SUPPLY]] entre ellos. Cuándo no existan dichas líneas, establecer el  [[encyclopedia FOCUS_TITLE]] a investigación puede resultar más eficiente para un planeta que otras opciones.
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
5: Todos los sistemas comparten puntos de [[metertype METER_RESEARCH]] (PI) que generan incluso sin tener líneas de [[metertype METER_SUPPLY]] entre ellos. Cuándo no existan dichas líneas, establecer el [[encyclopedia FOCUS_TITLE]] a investigación puede resultar más eficiente para un planeta que otras opciones.
```

We've got a problem here. We need to decide how to translate Research and Influence. If both end up starting with an "in", then we can't use PI or PIn.

File: default/stringtables/es.txt [hunk 7]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_06
+6: Las líneas de [[metertype METER_SUPPLY]] pueden ser usadas para repostar [[metertype METER_FUEL]] de las naves.
+
+BEGINNER_HINT_07
+7: Algunos edificios, como el [[buildingtype BLD_INDUSTRY_CENTER]], afectarán a cada planeta que tenga una conexión de [[metertype METER_SUPPLY]]. Otros, como [[buildingtype BLD_GAS_GIANT_GEN]], sólo afectan al sistema en el que se construyeron. Revise la sección [[encyclopedia ENC_BUILDING_TYPE]] para conocedr las funciones que cumplen los distintos edificios.
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
7: Algunos edificios, como el [[buildingtype BLD_INDUSTRY_CENTER]], afectarán a cada planeta que tenga una conexión de [[metertype METER_SUPPLY]]. Otros, como [[buildingtype BLD_GAS_GIANT_GEN]], sólo afectan al sistema en el que se construyeron. Revise la sección [[encyclopedia ENC_BUILDING_TYPE]] para conocer las funciones que cumplen los distintos edificios.
```

File: default/stringtables/es.txt [hunk 8]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_09
+9: Una [[predefinedshipdesign SD_COLONY_SHIP]] tarda más en construirse que el resto de [[predefinedshipdesign SD_OUTPOST_SHIP]]s, pero pueden colonizar planetas fuera de su rango de [[metertype METER_SUPPLY]].
+
+BEGINNER_HINT_10
+10: Con un sistema seleccionado en la [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]], clique con el botón derecho en la imagen de un planeta en el panel del sistema para acceder al reporte de idoneidad planetaria. Esto informará de qué especies del imperio podrán establecerse exitosamente en dicho planeta. Un número en verde indica la [[metertype METER_POPULATION]] máxima que el planeta podría alcanzar si lo colonizase una especie, mientras que un número rojo indica que la especie moriría si intentase colonizar dicho planeta.
+
+BEGINNER_HINT_11
+11: La [[metertype METER_POPULATION]] máxima alcanzable por un planeta puede ser aumentada investigando varias tecnologías de creciciemto. Algunos especiales inusuales pueden incrementar la [[metertype METER_TARGET_POPULATION]] para especies con metabolismos específicos, consulte [[encyclopedia GROWTH_FOCUS_TITLE]] para más información sobre estos especiales.
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
11: La [[metertype METER_POPULATION]] máxima alcanzable por un planeta puede ser aumentada investigando varias tecnologías de crecimiento. Algunos especiales inusuales pueden incrementar la [[metertype METER_TARGET_POPULATION]] para especies con metabolismos específicos, consulte [[encyclopedia GROWTH_FOCUS_TITLE]] para más información sobre estos especiales.
```

File: default/stringtables/es.txt [hunk 9]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_12
+12: Presta atención a los planetas con [[encyclopedia ENC_SPECIAL]], habitualmente indican la presencia de algún recurso especial que puede beneficiar a tu imperio si lo utilizas de manera apropiada ¡Pulsa botón derecho sobre el icono del recurso especial para acceder a su página de la Pedi y descubrir más! Right click on a special's icon to access its 'Pedia entry and find out more!
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
12: Presta atención a los planetas con [[encyclopedia ENC_SPECIAL]]es, habitualmente indican la presencia de algún recurso especial que puede beneficiar a tu imperio si lo utilizas de manera apropiada. Clica con el botón derecho sobre el icono del recurso especial para acceder a su página de la Pedia y descubrir más.
```

That "!" in the original English is quite out of place.

File: default/stringtables/es.txt [hunk 10]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_13
+13: Únicamente los [[species SP_EXOBOT]] pueden colonizar cinturones de asteroides. Si descubres muchos cinturones de asteroides próximos, ¡Puede que quieras investigar como construirlos más pronto que tarde!
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
13: Únicamente los [[species SP_EXOBOT]]s pueden colonizar cinturones de asteroides. Si descubres muchos cinturones de asteroides próximos, puede que quieras investigar como construirlos más pronto que tarde.
```

File: default/stringtables/es.txt [hunk 11]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_14
+14: Investiga mejoras en [[metertype METER_SUPPLY]],  como [[tech CON_ORBITAL_CON]], para incrementar la distancia que alcanzan tus lineas de suministro desde tu sistema.
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
14: Investiga mejoras en [[metertype METER_SUPPLY]], como [[tech CON_ORBITAL_CON]], para incrementar la distancia que alcanzan tus lineas de suministro desde tu sistema.
```

File: default/stringtables/es.txt [hunk 12]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_15
+15: La mayoria de las especies comienzan con un [[metertype METER_HAPPINESS]]  de 0 cuando su planeta es capturado, que - usualmente - irá aumentando en 1 por cada turno que pase. ¡Presta atención a las especies [[encyclopedia XENOPHOBIC_SPECIES_TITLE]]s! No les gusta estar cerca de otras especies, lo cual afectará a la estabilidad u otros valores tanto suyos como de especies cercanas.
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
15: La mayoria de las especies comienzan con una [[metertype METER_HAPPINESS]] de 0 cuando su planeta es capturado, que (usualmente) irá aumentando en 1 por cada turno que pase. Presta atención a las especies [[encyclopedia XENOPHOBIC_SPECIES_TITLE]]s. No les gusta estar cerca de otras especies, lo cual afectará a la estabilidad u otros valores tanto suyos como de especies cercanas.
```

File: default/stringtables/es.txt [hunk 13]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_16
+16: Puede que quieras retrasar la investigación de [[tech LRN_PSIONICS]] hasta que tu imperio incluya especies con [[encyclopedia TELEPATHIC_TITLE]].
+
+BEGINNER_HINT_17
+17: [[metertype METER_SHIELD]] provee defensa a una nave contra todo disparo contra la misma, reduciendo el daño recibido en el valor del escudo. Los valores de escudo de una nave pueden verse afectados por varios factores, como tormentas solares, en qué tipo de sistema estelar se encuentra la nave, o si estás cerca una capital imperial capturada.
+
+BEGINNER_HINT_18
+18: [[encyclopedia FOCUS_TITLE]] - presta atención al medidor de enfoque de los nuevos planetas que añadas a tu imperio, tanto colonizados como conquistados. Este medidor controla si el planeta se enfoca en la producción industrial, en investigación o en defensa. Avanzada la partida, más opciones de enfoque pueden ser desbloqueadas a medida que se investigan nuevas [[encyclopedia ENC_TECH]]s. Distintas especies  pueden tener disferentes opciones de enfoque disponibles.
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
18: [[encyclopedia FOCUS_TITLE]]: presta atención foco planetario de los nuevos planetas que añadas a tu imperio, tanto colonizados como conquistados. El foco seleccionado controla en que se concentra la producción: en industria, en investigación, en influencia, en defensa, etc. Avanzada la partida, más opciones de foco pueden ser desbloqueadas a medida que se investigan nuevas [[encyclopedia ENC_TECH]]s. Distintas especies pueden tener diferentes opciones de foco disponibles.
```

File: default/stringtables/es.txt [hunk 14]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_19
+19: Se puede construir una colonia en un [[encyclopedia OUTPOSTS_TITLE]] construyendo un edificio colonial o mediante una [[predefinedshipdesign SD_COLONY_SHIP]]. La mayoría de edificios coloniales requieren una conexión de [[metertype METER_SUPPLY]] a un planeta con un medidor de [[metertype METER_HAPPINESS]] de al menos 5, y una [[metertype METER_POPULATION]] de 3. Cuando se desbloquea una especie, como al investigar [[tech PRO_EXOBOTS]], no se aplica este requisito para su edificio colonial.
+
+BEGINNER_HINT_20
+20: Informes de situación concretos (como este) pueden ser temporalmente ignorados haciendo click con el botón derecho del ratón en su icono y seleccionando una de las opciones disponibles. Ciertos tipos de reportes pueden ser filtrados de manera permanente usando el menú de filtros en la parte inferior de esta pantalla . Estas pistas iniciales están agrupadas como [[encyclopedia BEGINNER_HINTS]] en la lista de filtros.
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
20: Informes de situación concretos (como este) pueden ser temporalmente ignorados haciendo clic con el botón derecho del ratón en su icono y seleccionando una de las opciones disponibles. Ciertos tipos de reportes pueden ser filtrados de manera permanente usando el menú de filtros en la parte inferior de esta pantalla. Estas pistas iniciales están agrupadas como [[encyclopedia BEGINNER_HINTS]] en la lista de filtros.
```

File: default/stringtables/es.txt [hunk 15]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_21
+21: Puedes fijar un destino para una nave mientras está todavía en la cola de producción. Con la pantalla de producción abierta: haz click en el sistema destino en el mapa galáctico y haz click con el botón derecho del ratón sobre la nave en la cola de producción. Cuando aparezca el menú con las distintas opciones elige 'Ir a' y entonces debería aparecer el nombre del sistema seleccionado al que quieres que viaje la nave tan pronto como esté construida.
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
21: Puedes fijar un destino para una nave mientras está todavía en la cola de producción. Con la pantalla de producción abierta: haz clic en el sistema destino en el mapa galáctico y haz click con el botón derecho del ratón sobre la nave en la cola de producción. Cuando aparezca el menú con las distintas opciones, elige 'Ir a' y entonces debería aparecer el nombre del sistema seleccionado al que quieres que viaje la nave tan pronto como esté construida.
```

File: default/stringtables/es.txt [hunk 16]
```diff
@@ -7362,127 +7362,128 @@ El imperio %empire% ha sido eliminado.
...
+
+BEGINNER_HINT_22
+22: Seleccionar un systema y pulsar ALT-C proporciona un círculo de rango para ese sistema, haz zoom para comparar la distancia mirando el tamaño del medidor de distancia.
```
Comment (Oberlus) [inline default/stringtables/es.txt]:
```suggestion
22: Seleccionar un sistema y pulsar ALT-C proporciona un círculo de rango para ese sistema, haz zoom para comparar la distancia mirando el tamaño de la barra de escala en la esquina superior izquierda del mapa.
```

I'm deliverately improving this hint for the Spanish version (the scale bar is not a distance meter indeed).

**Review by Oberlus — CHANGES_REQUESTED** [review]

**Comment by Oberlus** [comment]
I just found the "insert suggestion" option :dancing_men:

**Comment by rsf92** [comment]
I squashed all the suggested commits @Oberlus  and fixed the Focus thing on my own on a separate commit

**Comment by Oberlus** [comment]
> I squashed all the suggested commits @Oberlus and fixed the Focus thing on my own on a separate commit

There are a few suggested corrections pending, like this one:

https://github.com/freeorion/freeorion/pull/3680/files#r795063339

Check out for "creciciemto", "conocedr"...

**Comment by rsf92** [comment]
Okay, now,  I think

**Review by Oberlus — APPROVED** [review]

**Pull request merged**